### PR TITLE
fix(sdk-updates): Analytics was being rejected due to organizationId key

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

**Review by maheskett — APPROVED** [review]

**Review by scefali — APPROVED** [review]

File: static/app/components/globalSdkUpdateAlert.tsx [hunk 1]
```diff
@@ -45,21 +45,21 @@ const recordAnalyticsSeen = ({organization}: AnalyticsOpts) =>
   trackAnalyticsEvent({
     eventKey: 'sdk_updates.seen',
     eventName: 'SDK Updates: Seen',
-    organizationId: organization.id,
+    organization_id: organization.id,
```
Comment (wmak) [inline static/app/components/globalSdkUpdateAlert.tsx:48]: shouldn't this be
```suggestion
    org_id: organization.id,
```

based on what's in reload?
https://github.com/getsentry/reload/blob/master/reload_app/events.py#L988-L996

Comment (EvanPurkhiser) [reply static/app/components/globalSdkUpdateAlert.tsx:48]: This is kind of dumb but.. https://github.com/getsentry/getsentry/blob/e4e77129fc2e04ad9fdf93b496ba98bb852c750b/static/getsentry/gsApp/utils/trackAnalyticsEvent.tsx#L81-L83

Comment (wmak) [reply static/app/components/globalSdkUpdateAlert.tsx:48]: 🙈 welp...

**Review by wmak — APPROVED** [review]
lgtm

**Review by dashed — APPROVED** [review]

**Pull request merged**